ui/tests/unit/models/capabilities-test.js

```diff
@@ -81,4 +81,18 @@ module('Unit | Model | capabilities', function(hooks) {
     assert.notOk(model.get('canDelete'));
     assert.notOk(model.get('canList'));
   });
+
+  test('it does not require sudo on sys/leases/revoke if update capability is present', function(assert) {
+    let model = run(() =>
+      this.owner.lookup('service:store').createRecord('capabilities', {
+        path: 'sys/leases/revoke',
+        capabilities: ['update', 'read'],
+      })
+    );
+    assert.ok(model.get('canRead'));
+    assert.notOk(model.get('canCreate'), 'sudo requires the capability to be set as well');
+    assert.ok(model.get('canUpdate'), 'should not require sudo if it has update');
+    assert.notOk(model.get('canDelete'));
+    assert.notOk(model.get('canList'));
+  });
```

Comment:
Can we add another test for the paths in the SUDO_PREFIX arg too to make sure they _are_ requiring sudo?

So for `path: 'sys/leases/revoke-prefix/aws'` with capabilities "update", `canUpdate` should be false, but with "update", "sudo", `canUpdate` should be true.

Reply:
I added a couple. Do they look right?

Reply:
Yassss - perfect!